docs/how-to-use-flatbuffers.md

```diff
@@ -19,9 +19,3 @@ We recommend building `flatc` from source. To generate rust code, use
 ```console
 just gen-all-fbs-rust-code
 ```
-
```

Comment:
Can you update the "last version as of ..." date at the top of this file?

Reply:
done